recipe/style.css

```
@@ -0,0 +1,82 @@
+body{
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
+  min-height: 100vh;
+  background-image: url("img/dish.jpg");
+  background-size: cover;
+  margin: 0;
+  padding: 25px;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 1.5;
+  color: #333;
+}
+
+.section-blocks {
+  background-color: rgba(255,255,255,0.8);
+  padding: 15px 30px;
+  border-radius: 10px;
+  box-shadow: 2px 2px 5px rgba(0,0,0,0.2);
+  width: 30%;
+  max-width: 400px;
+}
+
+@media (max-width: 996px) {
+  .section-blocks {
+    margin: 0 auto 15px;
+    width: 100%;
+  }
+}
+
+.top-section {
+  display: flex;
+  justify-content: space-between;
+}
+
+@media (max-width: 996px) {
+  .top-section {
+    flex-direction: column;
+    margin-bottom: 15px;
+  }
+}
+
+.recipe-ingredients ul {
+  padding-left: 25px;
+}
+
+.recipe-ingredients ul li {
+  margin-bottom: 8px;
+}
+
+.recipe-ingredients ul li:last-of-type {
+  margin-bottom: 0;
+}
+
+.recipe-ingredients ul li img {
+  position: relative;
+  top: 6px;
+  max-width: 25px;
+}
+
+.recipe-instructions {
+  display:block;
```

```suggestion
  display: block;
```
Remember about consistent code formatting